src/ResourceManagement/Sql/SqlManagement/Generated/Models/EncryptionProtectorProperties.cs

```diff
@@ -41,15 +41,15 @@ public string ServerKeyName
             set { this._serverKeyName = value; }
         }
         
-        private string _type;
+        private string _serverKeyType;
         
         /// <summary>
         /// Optional. The Transparent Data Encryption Encryption Protector Type
         /// </summary>
-        public string Type
+        public string ServerKeyType
```

Comment:
We made breaking changes to the API. The feature is just preparing to enter private preview, so we thought it was acceptable to make the changes without needing to bump API version, since there will be no customer impact.